File: pgm/ABC_Kotlin/ABC_370_A.kt
// import java.time.*

// import kotlin.math.*

const val Yes: String = "Yes"
const val No: String = "No"
const val Invalid: String = "Invalid"
const val space: String = " "
const val one: Int = 1
const val zero: Int = 0

fun main() {
    val (L: Int, R: Int) = readln().split(space).map { it.toInt() }
    if (L == one && R == zero) {
        println(Yes)
    } else if (L == zero && R == one) {
        println(No)
    } else {
        println(Invalid)
    }
    kotlin.system.exitProcess(0)
}

File: pgm/ABC_Dart/ABC_370_A.dart
// import 'dart:collection';
import 'dart:io';
// import 'dart:math';

void main() {
  final [int L, int R] = stdin
      .readLineSync()!
      .split(' ')
      .map(int.parse)
      .toList();
  const String Yes = 'Yes';
  const String No = 'No';
  const String Invalid = 'Invalid';
  const int one = 1;
  const int zero = 0;
  if (L == one && R == zero) {
    print(Yes);
  } else if (L == zero && R == one) {
    print(No);
  } else {
    print(Invalid);
  }
  exit(0);
}
